### PR TITLE
Put the python lcmtypes into a new "lcmtypes" package

### PR DESCRIPTION
**Pull request by rdeits**
As per @patmarion's suggestion in #1373 

This modifies the lcmtypes.cmake file to put all python bindings into the `lcmtypes` package on install. The result is that the drake lcmtypes are now accessible as `lcmtypes.drake` instead of just `drake`, which frees up the `drake` name to be used for something more general.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -84,13 +84,13 @@ set(bullet_GIT_REPOSITORY https://github.com/RobotLocomotion/bullet-pod.git)
 set(bullet_GIT_TAG 4319ffd7e9251066d93064f5a8dab12e33dbe5e2)
 set(bullet_IS_PUBLIC TRUE)
 set(bullet_IS_CMAKE_POD TRUE)
-set(cmake_GIT_REPOSITORY https://github.com/RobotLocomotion/cmake.git)
-set(cmake_GIT_TAG 3ca531594097919f0c9784585ecb32fe0df15404)
+set(cmake_GIT_REPOSITORY https://github.com/rdeits/cmake.git)
+set(cmake_GIT_TAG 25149772a2734589d3a925bc62f92ba07e6f3a97)
```
Comment (RussTedrake) [inline CMakeLists.txt]: i've merged.   we should decide whether accepting local forks is ok or not.  there is a risk that your branch could disappear leaving the main repo hanging.

Comment (rdeits) [inline CMakeLists.txt]: I think that probably the right procedure is:

1. open pull request in the external
1. open pull request in drake pointing to fork
1. when drake pull request passes review & tests, then merge the external pull request and update drake to point to master on the external
1. wait for one more round of unit tests on drake

**Comment by rdeits** [comment]
Okay, I've updated this to point to cmake master

**Comment by RussTedrake** [comment]
i believe director was merged, so just waiting on @rdeits to change that git tag back, too.

**Comment by patmarion** [comment]
yep. @rdeits you should leave the git tag as is (ce35723e54707787c5e31872231abefd93cf918f) but revert the repo url to RobotLocomotion/director.git

**Comment by rdeits** [comment]
okay, done

**Pull request merged**